File: problemas/ad-hoc/1546/1546.c
#include <stdio.h>

int main(){
    int N, K, feedback;

    scanf("%d", &N);
    
    for(int i = 0; i < N; ++i){
        scanf("%d", &K);

        for(int j = 0; j < K; ++j){
            scanf("%d", &feedback);

            switch(feedback){
                case 1: printf("Rolien\n");
                        break;
                case 2: printf("Naej\n");
                        break;
                case 3: printf("Elehcim\n");
                        break;
                case 4: printf("Odranoel\n");
                        break;
            }
        }
    }

    return 0;
}

File: problemas/ad-hoc/1546/1546.cs
using System;

class URI {
    static void Main(string[] args) {
        string[] responsaveis = new string[] {"Rolien", "Naej", "Elehcim", "Odranoel"};
        
        int N = int.Parse(Console.ReadLine());
        
        for(int k = 0; k < N; ++k){
            int K = int.Parse(Console.ReadLine());
            
            for(int i = 0; i < K; ++i){
                int feedback = int.Parse(Console.ReadLine());
                Console.WriteLine(responsaveis[feedback - 1]);
            }
        }
    }
}